tskv/src/record_file/record.rs

```diff
@@ -0,0 +1,5 @@
+pub struct Record {
```

Comment:
why not define Record struct on mod.rs ?

Reply:
Temporarily reserved